app/src/main/java/com/currencyconverter/data/models/Country.kt

```diff
@@ -0,0 +1,22 @@
+package com.currencyconverter.data.models
+
+class CountryCurrency {
```

Comment:
Models could be implemented as data classes.

E.g. 
`data class Country(val code: String, val name: String)`